src-packed/textAnalytics.js

```diff
@@ -80,9 +80,18 @@ function removeCodeBlocks (text) {
     return result;
 }
 
+// Replace backticks and single quotes for double quotes
+// since backticks and single quotes make sentiment more negative
+function replaceQuotes (text) {
+    var result = text.replace(/`/g, "\"");
+    result = result.replace(/\'/g, "\"");
```

Comment:
You know, I think I will take out the `'` replacement here.

Your comment on contractions made me thing this might break something -- changing `won't` to `won"t`, for example.